guides/rails_guides/markdown/renderer.rb

```diff
@@ -58,18 +66,16 @@ def convert_footnotes(text)
           end
         end
 
-        def class_for(code_type)
+        def lexer_language(code_type)
           case code_type
-          when "ruby", "sql", "plain", "js", "yaml"
+          when "js", "http", "html", "ruby", "sql", "yaml", "yml"
```

Comment:
One last nitpick, "http" is unused:

```suggestion
          when "js", "html", "ruby", "sql", "yaml", "yml"
```

Reply:
I've removed `http`.
I've also removed `yml` because it was used only once, and I've replaced the code block to use `yaml` instead.

Reply:
Sorry, something went wrong with pushing changes. I've pushed the above mentioned changes.